### ISSUE TITLE
Is it possible to remove the auto generated header at the top of the markdown file?

### ISSUE THREAD

**Issue by pbacon-blaber** [Mermade/widdershins#230]
I was wondering if it's possible to remove the "header" that's getting generated at the top of the output markdown file?

```
---
title: my_app
language_tabs:
  - shell: Shell
  - http: HTTP
  - javascript: JavaScript
  - javascript--nodejs: Node.JS
  - ruby: Ruby
  - python: Python
  - java: Java
  - go: Go
toc_footers: []
includes: []
search: true
highlight_theme: darkula
headingLevel: 2

---
```

I'm using custom user templates as I just would like to generate a plain markdown file, but haven't been able to figure out how to remove this from the top.

**Comment by MikeRalphson** [comment]
The `master` version has (and the next release will have) an `omitHeader` option for this very purpose.

**Comment by samueljseay** [comment]
Hi, just wondering if a new release is on its way?

**Comment by tomkerkhove** [comment]
I'd love to have this as well

**Comment by tomkerkhove** [comment]
Just to give a reason, it's breaking our Slate rendering:
![image](https://user-images.githubusercontent.com/4345663/60820794-7cad0b00-a1a2-11e9-9d19-f1e755849340.png)

**Comment by MikeRalphson** [comment]
@tomkerkhove this is somewhat surprising, as the header is there and in that format because that's what Slate uses. (Link). 

How would features like language tabs work if the header were (routinely) omitted?

**Comment by tomkerkhove** [comment]
I'm not sure but something must be up qi think.

Did a rough test and omitting fixed it but didn't check code samples 🤔

I'm a slate rookie so not sure what to do

**Comment by tomkerkhove** [comment]
Our issue is that we were abusing includes which caused the funky result.

One option would be to move them to dedicated pages but then we lose the one-pager aspect of Slate. What do you advise here as we must not be the only ones wanting to achieve this.

[Deep nesting](https://github.com/lord/slate/wiki/Deeper-Nesting) sounds interesting where the tags + operations would be using a smaller header, but again depends on the generator.

**Comment by tomkerkhove** [comment]
Frankly, if we could use `--omitHeader` to remove the metadata header and use every API specification as an include that would help us tremendously.

That way we can have one page with all our documentation and it's up to us to ensure all code metadata is in the general page header.

Unless there is a better approach you'd suggest?

**Comment by MikeRalphson** [comment]
I hadn't planned it for that, but if you're generating includes using Widdershins, then --omitHeader will be the way to go.

**Comment by tomkerkhove** [comment]
Are there plans to ship a new version soon? As `3.6.7` does not include this yet.

Thanks for all the help!

**Comment by MikeRalphson** [comment]
Yes as soon as #223 is fixed.

**Comment by tomkerkhove** [comment]
Sounds good, thanks!

**Comment by MikeRalphson** [comment]
Should be fixed in version `4.0.0`, which includes breaking changes (please see the `README.md` file).
